src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -103,6 +102,11 @@ object Coroutine {
     var $yield: Y = null.asInstanceOf[Y]
     var $result: R = null.asInstanceOf[R]
 
+    /**
+      * Clones the coroutine that this instance is a part of
+      * @return A new instance with the same type arguments, stacks, and internal
```

Comment:
Separate param and return section from the rest of the comment with an empty comment line.